src/eos-install-app-helper-installer.py

```diff
@@ -229,6 +254,16 @@ def _run_app_center_for_app(self, app_id, remote, app_name):
         except OSError as e:
             exit_with_error("Could not launch {}: {}".format(app_name, repr(e)))
 
+        self._wait_for_installation(app_id, app_name)
+
+        if not self._check_app_flatpak_launcher(app_id, app_name):
+            exit_with_error("{} isn't installed - something went wrong in GNOME Software".format(app_name))
+
+        logging.info("{} successfully installed".format(app_name))
```

Comment:
I think I'd rather keep this actually - just because we'll know if it actually succeeded or if it just hung. The logs go to the journal, which I believe is included in diagnostic output users can generate.